sound/soc/sof/mediatek/mt8195/mt8195.c

```diff
@@ -316,6 +316,20 @@ static int mt8195_dsp_suspend(struct snd_sof_dev *sdev, u32 target_state)
 {
 	struct platform_device *pdev = container_of(sdev->dev, struct platform_device, dev);
 	int ret;
+	int reset_sw, dbg_pc, sleep_try = 2000, sleep_cnt = 0;
```

Comment:
commit message issues
nodify->notify

there is a race condition.

too many negatives: not to ensure that turn off adsp clock